functests/4_latency/latency.go

```diff
@@ -61,8 +61,12 @@ var _ = Describe("[performance] Latency Test", func() {
 	var profile *performancev2.PerformanceProfile
 	var latencyTestPod *corev1.Pod
 	var err error
+	var logName string
+	var logFileContent string
```

Comment:
Why exactly do we need this one?

Reply:
to avoid using calling getLogFile(..) inside the gomega Expect, according to @fromanirh we need to call it & handle it explicitly & not nest it like it was .